### PR TITLE
Task/temp 136 global build number

### PR DESCRIPTION
**Pull request by minimusic**
Moved the build number and re-generated template. Testing now.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -157,6 +157,11 @@ commands:
       - restore_cache:
           key: 2-gems-{{ checksum "Gemfile.lock" }}
       - run: bundle check || bundle install
+      - run:
+          name: Update Gemfile
+          command: |
+            cd PRODUCTNAME
```
Comment (ZevEisenberg) [inline .circleci/config.yml]: I don’t think you need to `cd` first. The Gemfile should be active at the top level of the repo, right?

Comment (minimusic) [reply .circleci/config.yml]: In case the build script gets re-ordered in the future, isn't it safer to keep it?

Comment (ZevEisenberg) [reply .circleci/config.yml]: I believe Bundler looks recursively up until it finds a parent folder with a Gemfile. Should be fine anywhere in the repo.

Comment (minimusic) [reply .circleci/config.yml]: hm... we've got the `- run: bundle check || bundle install` right above which is running at the parent path, so the goal here was to drill into the child folder, not up to a parent folder. I don't see any option for `bundle install` to check/update all subdirectories, so I think we are stuck with the `cd` unless we want to write some recursive code here to walk the folder tree.

Comment (ZevEisenberg) [reply .circleci/config.yml]: Ohhhh, I totally forgot we had this requirement because this is the template project, and everything is meta. As you were 😅

**Pull request merged**